modules/gcve-private-cloud/variables.tf

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "description" {
+  description = "Private cloud description."
+  type        = string
+  default     = "Terraform-managed."
+}
+
+variable "management_cidr" {
+  description = "vSphere/vSAN subnets CIDR range."
+  type        = string
+}
+
+variable "management_cluster_config" {
+  description = "Management cluster configuration."
+  type = object({
+    node_type_id      = string
+    node_count        = number,
+    custom_core_count = number
+  })
+  default = {
+    node_type_id      = "standard-72",
+    node_count        = 3,
+    custom_core_count = null
+  }
+}
+
+variable "name" {
+  description = "Private cloud name."
+  type        = string
+}
+
+variable "private_connections" {
+  description = "VMWare private connections configuration. It is used to create the gcloud command printed as output."
+  type = map(object({
+    name              = string
+    network_self_link = string
+    peering           = string
```

Comment:
nit: I would rename this to `peering_name`

Reply:
it's an object with `name` as attribute, maybe `psa_peering`?

Reply:
fixed

Reply:
ach, agree `peering_name` is the best choice, I thought we were discussing the variable not the attribute